vector/src/main/java/lt/lrv/basemap/layers/Waterway.java

```diff
@@ -43,17 +43,26 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
                 addWaterwayLine(FieldValues.CLASS_RIVER, 10, sf, features);
             } else if (List.of("hc31", "hc32").contains(code) && (type == 3 || type == 4)) {
                 addWaterwayLine(FieldValues.CLASS_DITCH, 11, sf, features);
+            } else {
+                addWaterwayLine(FieldValues.CLASS_STREAM, 12, sf, features);
             }
         }
     }
 
     void addWaterwayLine(String clazz, int minZoom, SourceFeature sf, FeatureCollector features) {
         var length = (int) sf.getLong("PLOTIS");
         var name = nullIfEmpty(sf.getString("VARDAS"));
+        var code = sf.getString("GKODAS");
+
+        var brunnel = switch (code) {
+            case "op01", "hc1op0", "hc31op0", "hc32op0", "hc33op0", "hc3op0" -> "tunnel";
```

Comment:
You can use `FieldValues.BRUNNEL_TUNNEL` instead of `tunnel` string. This way you will avoid typos.